familyTree.py

```diff
@@ -26,14 +26,19 @@
 # person (this should be the empty list if there are none). The order of the list
 # does not matter and duplicates will be ignored.
 
-def ancestors(genealogy, person, lst = []):
+def ancestors(genealogy, person):
     # your code here
+    ancestors_list = []
     if genealogy.__contains__(person):
-        lst.append(genealogy[person][0])
-        lst.append(genealogy[person][1])
-        ancestors(genealogy, genealogy[person][0], lst)
-        ancestors(genealogy, genealogy[person][1], lst)
-        return lst
+        ancestors_list.append(genealogy[person][0])
```

Comment:
The functions that start and end with double underscore are not explicitly called normally. A key in a dict can be check by simply. 
```
if person in genealogy:
    # do something
```